### PR TITLE
Fix names (use date instead of revs) and update pkgs (cont)

### PR DESCRIPTION
**Pull request by jagajaga**
So see https://github.com/NixOS/nixpkgs/pull/6376
(I've rebased agains master and previous PR closed :cry:)
cc @iElectric @edolstra @lethalman @vcunat

### CODE REVIEW

File: pkgs/applications/audio/distrho/default.nix [hunk 1]
```diff
@@ -1,16 +1,13 @@
 { stdenv, fetchgit, alsaLib, fftwSinglePrec, freetype, jack2
 , libxslt, lv2, pkgconfig, premake3, xlibs }:
 
-let
-  rev = "3bfddf7f";
-in
 stdenv.mkDerivation rec {
-  name = "distrho-${rev}";
+  name = "DISTRHO-Ports-git-2015-01-28";
```
Comment (vcunat) [inline pkgs/applications/audio/distrho/default.nix]: Thinking of it now, upper case should be avoided in names. http://nixos.org/nixpkgs/manual/#idm140737319157136

**Comment by jagajaga** [comment]
@vcunat my fault, will fix. But what about documenting all this stuff?

**Comment by bjornfor** [comment]
@jagajaga: It is documented, see the above link from @vcunat :-)

**Comment by jagajaga** [comment]
@bjornfor I can't see something about dates of commits in names when using git.

**Comment by bjornfor** [comment]
But this wasn't about dates, it was about lower-case package names.

Were you thinking of whether to use dashes or in date strings? Yeah, that should probably be agreed upon and documented.

**Comment by jagajaga** [comment]
I've already apologized about upper case :)

Maybe I should write about dates to ML?
Why I think it's better to use dashes (like YYYY-MM-DD), because it's more friendly to be read, IMO.

**Comment by iElectric** [comment]
Let's doxument the dates in manual guidelines

**Pull request merged**